src/Controls/tests/UITests/Tests/Issues/Issue18613.cs

```diff
@@ -0,0 +1,38 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.AppiumTests.Issues
+{
+	public class Issue18613 : _IssuesUITest
+	{
+		public Issue18613(TestDevice device) : base(device) { }
+
+		public override string Issue => "Disabled Button cannot trigger Command";
+
+		[Test]
+		public void CommandDisableButton()
+		{
+			App.WaitForElement("WaitForStubControl");
+
+			// Appium cannot find a disable Button using FindElement
+			if (Device != TestDevice.iOS && Device != TestDevice.Mac)
```

Comment:
Can you add a hidden button below the button? Or use something else that can be found? I did this with my InputTransparent tests and it appears that tapping a element just uses the center coordinates.

I also made that bottom button fail the test so if something goes horribly wrong and the tap passes through because the button is gone, then we know.

Reply:
We have a similar situation. We only look for the button to try to click it. Even if we don't do it on iOS, by verifying the snapshot validating the visual state of the button, we can know if the button is disabled.